assets/src/domain/eventEditor/services/filterState/datetimes/types.ts

```diff
@@ -0,0 +1,44 @@
+import {
+	SortBy,
+	EntityFilterState,
+	EntityFilterAction,
+	EntityFilterActionType,
+	EntityFilterStateManager,
+	EntityFilterStateReducer,
+} from '@sharedServices/filterState';
+import { EntityListFilterStateManager } from '@appLayout/entityList/filterBar';
+
+export interface DatetimesFilterState extends EntityFilterState {
+	datetimesToShow: DatetimesToShow;
+}
+
+export type DatetimesFilterActionType = 'SET_DATETIMES_TO_SHOW' | EntityFilterActionType;
+
+export interface DatetimesFilterAction
+	extends Partial<DatetimesFilterState>,
+		EntityFilterAction<DatetimesFilterActionType> {}
+
+export interface DatetimesFilterStateManager
+	extends EntityListFilterStateManager<SortBy>,
```

Comment:
not sure what this `<SortBy>` does?

Reply:
It sets the proper union types that you pass to `sortBy()`. Application `SortBy` is only `'name' | 'id'`. If a domain wants more `SortBy` options, it should be passed here. I could leave it as string but you know that makes it prone to mistakes.